lib/chess/core/record.rb

```diff
@@ -68,6 +34,34 @@ def current_record
       def last_record_available?
         @history.last[@current_record_team].nil?
       end
+
+      def add_move_record(accomplished_intention, capture)
+        piece = accomplished_intention.target_cell.occupant.symbol
```

Comment:
Try renaming these local variables with more verbose names, like this
```suggestion
        occupant_symbol = accomplished_intention.target_cell.occupant.symbol
```
It's not a piece, because it's not an instance of the `BasePiece` class or any of its childs

Reply:
done!